recipes/perl-vcftools-vcf/0.840/run_test.pl

```diff
@@ -0,0 +1 @@
+use Vcf;
```

Comment:
I tried, but with "imports" simply "run_test.py" is called which apparently then tries to "import Vcf" in Python. 

Is there an explicit way to tell the interpreter to try the a default "run_test.pl"? Is that documented somewhere?